integration_tests/src/main/python/udf_cudf_test.py

```diff
@@ -0,0 +1,283 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import pandas as pd
+import time
+from typing import Iterator
+from pyspark.sql import Window
+from pyspark.sql.functions import pandas_udf, PandasUDFType
+from spark_session import with_cpu_session, with_gpu_session
+from marks import udf, allow_non_gpu
+
+_conf = {
+        'spark.rapids.sql.exec.ArrowEvalPythonExec':'true',
+        'spark.rapids.sql.exec.MapInPandasExec':'true',
+        'spark.rapids.sql.exec.FlatMapGroupsInPandasExec': 'true',
+        'spark.rapids.sql.exec.AggregateInPandasExec': 'true',
+        'spark.rapids.sql.exec.FlatMapCoGroupsInPandasExec': 'true',
+        'spark.rapids.sql.exec.WindowInPandasExec': 'true',
+        'spark.rapids.python.gpu.enabled': 'true'
+        }
+
+def _create_df(spark):
+    return spark.createDataFrame(
+            [(1, 1.0), (1, 2.0), (2, 3.0), (2, 5.0), (2, 10.0)],
+            ("id", "v")
+        )
+
+def _assert_cpu_gpu(cpu_func, gpu_func, cpu_conf={}, gpu_conf={}, is_sort=False):
+    print('### CPU RUN ###')
+    cpu_start = time.time()
+    cpu_ret = with_cpu_session(cpu_func, conf=cpu_conf)
+    cpu_end = time.time()
+    print('### GPU RUN ###')
+    gpu_start = time.time()
+    gpu_ret = with_gpu_session(gpu_func, conf=gpu_conf)
+    gpu_end = time.time()
+    print('### WRITE: GPU TOOK {} CPU TOOK {} ###'.format(
+        gpu_end - gpu_start, cpu_end - cpu_start))
+    print('### CPU RETURN ###')
+    print(cpu_ret)
+    print('### GPU RETURN ###')
+    print(gpu_ret)
```

Comment:
If this is not needed for debugging any longer it would be good to remove it.

Reply:
Done